### DISCUSSION TITLE
Received Token from ETH to SOL wallet, but as NFT

### DISCUSSION THREAD

**Discussion by maromandarino** [wormhole-foundation/wormhole#473] [Developer Q&A] [answered]
Hi, I used the wormhole bridge from my Metamask to transfer some SRM (ERC20) to Phantom.
When I redeemed them, apparently I can't spend them because are tokenized as NFT. 
Any idea to solve this issue?

Tx: https://explorer.solana.com/address/5Y9Z1kSe5tio1BxwtkYLLn7UBT2tPXaX9rRPkuvg1ceq

![image](https://user-images.githubusercontent.com/48258004/134484862-5e1fe5d9-8ed7-479e-a7e9-f0e870ef8d1d.png)

**Comment by leoluk** [comment]
Known Phantom bug, a release with a fix is pending.

**Reply by maromandarino** [reply]
Awesome. Thank you.

**Comment by BLACKoin** [comment]
@leoluk @maromandarino 
I have the same issue. What is the fix?
![ETH to SOL_NFT](https://user-images.githubusercontent.com/92128631/136477530-99edd3d3-56cc-4c44-a32c-cace253977ba.PNG)

ETH Txn: 0x9560685ee29f28e76b522c9338f080c27f67d1e184c0348ba76ac52f75dbf253
Sollana wallet address: PQJVRCJ7rtpvZXFXEWXCW9i91hN8p4hxoqeuHjiCrDP
sent to: AoyQUXh5GSnAmfvUTGTfLvRrazYCegpistkLL3MArwoj

Help!

**Comment by BLACKoin** [comment, marked as answer]
@maromandarino @leoluk 

Found a workaround.  

1) connect your wallet to the Orca dot so  DEX  (you can verify this is a common DEX for SOlLnan and not a scam).

2) ETH will show up as whETH

3) swap for SOL or another coin

![image](https://user-images.githubusercontent.com/92128631/136484075-afb02e19-5f5a-4167-a653-92efc02885a9.png)

**Reply by maromandarino** [reply]
Glad this was working for ETH. After the Phantom update I have a sort of 'Wrapped Wormhole SRM' in my wallet which I still can't exchange. lol
